modules/account-lib/test/unit/coin/eth/transactionBuilder/walletInitialization.ts

```diff
@@ -90,7 +90,7 @@ describe('Eth Transaction builder wallet initialization', function () {
       tx.type.should.equal(TransactionType.WalletInitialization);
       const txJson = tx.toJson();
       txJson.gasLimit.should.equal('1000');
-      txJson.gasPrice.should.equal('10');
+      txJson.gasPrice?.should.equal('10');
```

Comment:
also what is the reasoning behind all of these specifications?

Reply:
gasPrice is now optional but all the tests still assume legacy transaction, so i can either use the not null assertion `!` here or the `?`. I think i will go with the `!` and also add a check that the txn type is legacy.